src/ft2font_wrapper.cpp

```diff
@@ -1545,7 +1545,7 @@ PyMODINIT_FUNC PyInit_ft2font(void)
     FT_Int major, minor, patch;
     char version_string[64];
     FT_Library_Version(_ft2Library, &major, &minor, &patch);
-    sprintf(version_string, "%d.%d.%d", major, minor, patch);
+    snprintf(version_string, sizeof(version_string), "%d.%d.%d", major, minor, patch);
```

Comment:
This could have been PyOS_snprintf for consistency with the rest of the codebase.

Reply:
OK! It still complains, so can change it.